Resources/Locale/en-US/ghost/ghost-respawn.ftl

```diff
@@ -1,15 +1,15 @@
-ghost-respawn-time-left = Before the opportunity to return to the round { $time } 
-    { $time ->
-        [one] minute
-       *[other] minutes
-    }
-ghost-respawn-max-players = The function is not available, there should be fewer players on the server { $players }.
-ghost-respawn-window-title = Rules for returning to the round
-ghost-respawn-window-rules-footer = By using this feature, you [color=#ff7700]agree[/color] [color=#ff0000]not to transfer[/color] the knowledge of your past character to a new one. For violation of the clause specified here, [color=#ff0000]a ban in the amount of 3 days or more follows[/color].
-ghost-respawn-same-character = You cannot enter the round for the same character. Change it in the character settings.
+ghost-respawn-time-left = Please wait {$time} {$time ->
+    [one] minute
+   *[other] minutes
+} before trying to return to the round.
 
-ghost-respawn-log-character-almost-same = Player { $player } { $try ->
-    [true] join
+ghost-respawn-max-players = Cannot return to the round right now. There should be fewer players than {$players}.
```

Comment:
```suggestion
ghost-respawn-max-players = Cannot respawn right now. There should be fewer than {$players} players.
```

I assume this renders as "Cannot respawn right now. There should be fewer than 50 players." with my update.

I think my suggestion isn't perfect still, but it needs to be tweaked from what it is now.